.github/workflows/task-unit-test.yml

```diff
@@ -45,3 +49,6 @@ jobs:
         with:
           name: valgrind tests reports
           path: bin/Linux-x86_64-debug/unit_tests/Testing/Temporary/
+      - name: codecov
+        if: ${{ inputs.run-codecov }}
+        run: make coverage
```

Comment:
I think this one is reduendent, no?

Reply:
it defaults to false and might be useful in the future

Reply:
I don't like it. Workflows should be clean

Reply:
ok :(